src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEnum.cs

```diff
@@ -45,305 +45,1348 @@ public enum AccessibilityInterface
         EditableText = 2,
     }
 
+    /// <summary>
+    /// Accessibility reading information types.
+    /// </summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
     [Flags]
     public enum AccessibilityReadingInfoTypes : int
     {
+        /// <summary>
+        /// None.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         None = 0,
+        /// <summary>
+        /// Name trait for reading information.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         Name = 1,
+        /// <summary>
+        /// Role trait for reading information.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         Role = 2,
+        /// <summary>
+        /// Description trait for reading information.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         Description = 4,
+        /// <summary>
+        /// State trait for reading information.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         State = 8
     };
 
+    /// <summary>
+    /// Accessibility gesture types.
+    /// </summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
     public enum AccessibilityGesture
```

Comment:
It would be a great idea, but all enum values are used in screen-reader implementation.
So, legacy DALi (under 6.0 version) and EFL have same values as NUI.